src/image.rs

```diff
@@ -1184,6 +1206,87 @@ pub struct ProgressDetail {
     total: Option<u64>,
 }
 
+/// Describes query parameters for DELETE /images/{name}
+/// https://docs.docker.com/engine/api/v1.42/#tag/Image/operation/ImageDelete
+#[derive(Default, Debug)]
+pub struct DeleteOptions {
+
+    params: HashMap<&'static str, String>,
+}
+
+impl DeleteOptions {
+    /// Remove the image even if it is being used by stopped containers or has other tags
+    pub fn force(mut self) -> Self {
+        self.params.insert("force", true.to_string());
+        self
+    }
+
+    /// Do not delete untagged parent images
+    pub fn no_prune(mut self) -> Self {
```

Comment:
Should this be `noprune` (without an underscore) to match the actual parameter name in the Docker documentation?